packages/editor/src/lib/utils/licensing.ts

```diff
@@ -0,0 +1,117 @@
+import crypto from 'crypto'
+/*
+  Convert an ArrayBuffer into a string
+  from https://developer.chrome.com/blog/how-to-convert-arraybuffer-to-and-from-string/
+*/
+export function ab2str(buf: ArrayBuffer) {
+	return String.fromCharCode.apply(null, new Uint8Array(buf) as unknown as number[])
+}
+/*
+  Convert a string into an ArrayBuffer
+  from https://developers.google.com/web/updates/2012/06/How-to-convert-ArrayBuffer-to-and-from-String
+*/
+export function str2ab(str: string) {
+	const buf = new ArrayBuffer(str.length)
+	const bufView = new Uint8Array(buf)
+	for (let i = 0, strLen = str.length; i < strLen; i++) {
+		bufView[i] = str.charCodeAt(i)
+	}
+	return buf
+}
+
+export async function exportCryptoKey(key: CryptoKey, isPublic = false) {
+	const keyType = isPublic ? 'PUBLIC' : 'PRIVATE'
+	const exported = await crypto.subtle.exportKey(isPublic ? 'spki' : 'pkcs8', key)
+	const exportedAsBase64 = btoa(ab2str(exported))
+	return `-----BEGIN ${keyType} KEY-----\n${exportedAsBase64}\n-----END ${keyType} KEY-----`
+}
+
+/*
+  Generate a sign/verify key pair.
+*/
+export async function generateKeyPair() {
```

Comment:
let's move the functions that are just needed for Jest testing into the Jest file. we shouldn't have the functions here that won't be used in the client (to keep the bundle smaller). so, to be specific: `generateKeyPair`, `importPrivateKey`, `generateLicenseKey`

Reply:
moving this stuff out will help solve the crypto import thing